lib/host/gen_helpers.js

```diff
@@ -259,28 +253,12 @@ const genHelpers = (state = {}, adapter) => {
           if (attempt === MAX_SUBMIT_ATTEMPTS) {
             await state.ev.emit('error:evaluate_balance', gid, order, { text })
           } else {
-            const retryDelay = Math.max(500, delay + 250) // min 0.5s, increasing
-
-            await adapter.submitOrderWithDelay(connection, retryDelay, order)
+            await adapter.submitOrder(connection, order)
```

Comment:
should we wait a bit here? how about 500ms?

Reply:
this part was fixed in https://github.com/bitfinexcom/bfx-hf-algo/pull/121/commits/2543ced887c50a6734d355c99041461b48807e8c